modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/paths/PRMHomePage.path

```diff
@@ -12,12 +12,7 @@
 <tbody>
 <tr>
 	<td>CLICK_MDF</td>
-	<td>//div[contains(@class,'structure-item-container')]//a[contains(text(), 'MDF')]</td>
-	<td></td>
-</tr>
-<tr>
-	<td>ASSERT_REQUESTS</td>
-	<td>//div[@class='border-0 my-4']//h1[contains(text(),'${key_title}')]</td>
+	<td>//a[contains(@href,'mdf-requests')][contains(.,'MDF')]</td>
 	<td></td>
```

Comment:
Hi @juliosilveira77, As I said yesterday, will be better if you add two variables on this path, to be more generic.
The first can be `${key_buttonLink}` and the second `${key_buttonText}`. What do you think?